README.md

```diff
@@ -1,89 +1,162 @@
-SYCL-BLAS Implementation
+# SYCL-BLAS, a SYCL Basic Linear Algebra Library 
 ===
 
-SYCL-BLAS implements BLAS - [Basic Linear Algebra Subroutines](https://en.wikipedia.org/wiki/Basic_Linear_Algebra_Subprogram) - using [SYCL 1.2](
-https://www.khronos.org/registry/sycl/specs/sycl-1.2.pdf), the
-[Khronos](http://www.khronos.org) abstraction layer for [OpenCL](https://www.khronos.org/opencl/).
+The SYCL-BLAS library is an implementation of the 
+[Basic Linear Algebra Subroutines](https://en.wikipedia.org/wiki/Basic_Linear_Algebra_Subprogram) - using [SYCL 1.2.1](
+https://www.khronos.org/registry/sycl/specs/sycl-1.2.1.pdf) and is written 
+using C++11 features.
 
-SYCL-BLAS is a current work in progress research project from an ongoing
-collaboration with the *High Performance Computing & Architectures (HPCA) group*
-from the Universitat Jaume I [UJI](http://www.hpca.uji.es/).
+BLAS operations are being used to solve many complex problems in a range of areas 
+including science, imaging and AI.
 
-SYCL-BLAS is written using modern C++. The current implementation uses C++11
-features.
-See [Roadmap](Roadmap.md) for details on the current status and plans for
-the project.
-
-## Table of Contents
```

Comment:
Could you please keep the table of contents, it can help users to jump to the section needed to be

Reply:
I have added the table of contents back